spec/integration/elasticsearch/pipeline.spec.yml

```diff
@@ -48,6 +48,7 @@ spec:
         # set_security_user: false # Applicable to packages?
         sort: { type: object }
         split: { type: object }
+        terminate: { type: object }
```

Comment:
Btw, please add a patch below, so this processor is actually not available in older versions of the spec, something like this:
```
  - before: 3.3.0
    patch:
      - op: remove
        path: /definitions/processors/terminate # remove terminate processor validation
```